sass/navigation.scss

```diff
@@ -11,13 +11,28 @@
     font-size: 19px;
   }
 
+  h5 {
+    font-size: 16px;
+  }
+
   .navList {
     position: relative;
 
+    .navGroup.navGroupActive {
+      .subList {
+        display: block;
+      }
+
+      h4 > a {
+        text-transform: uppercase;
+      }
```

Comment:
Previously we had a sort of TITLE CASE thing going on: upper case, but smaller font and a different colour.

It's less obvious to me that the upper case section is active. How about reusing the hover style?

```suggestion
      a {
        background-color: #DADADA;
      }
```

Reply:
picture reference: 
![image](https://user-images.githubusercontent.com/5179225/136996234-e7d9d056-34cf-4b8c-a532-0df5d7a382cd.png)